README.md

```diff
@@ -737,7 +737,9 @@ Here are a few things to note when evaluating and using Bolt:
 * The data structures in the Bolt database are memory mapped so the data file
   will be endian specific. This means that you cannot copy a Bolt file from a
   little endian machine to a big endian machine and have it work. For most
-  users this is not a concern since most modern CPUs are little endian.
+  users this is not a concern since most modern CPUs are little endian. This
+  does not affect your code and you should always use big endian when converting
+  your keys and data to/from byte slice.
```

Comment:
Can you move this to a separate bullet point? I think it's unrelated to the endian encoding of the database. Also, it's not clear that the byte encoding is important for sorting specifically.

Maybe something like:

* If you are using integer keys you should use big endian encoding when converting to byte slices so that the keys are in byte sortable order. This allows cursors to iterate over the keys in the expected order.